src/radical/pilot/umgr/scheduler/backfilling.py

```diff
@@ -110,15 +111,15 @@ def update_pilots(self, pids):
 
                 state = self._pilots[pid]['state']
 
-              # self._log.debug('=== update pilot: %s %s', pid, state)
+              # self._log.debug('update pilot: %s %s', pid, state)
 
                 if  rps._pilot_state_value(state) < _BF_START_VAL:
-                  # self._log.debug('=== early')
+                  # self._log.debug('early')
```

Comment:
Maybe a bit more descriptive debug message?

Reply:
I'd rather remove them altogether I think - they are only needed for localized debugging...  Thanks!